src/uct/tcp/tcp.h

```diff
@@ -405,14 +405,20 @@ typedef struct uct_tcp_iface_config {
     uct_iface_mpool_config_t       tx_mpool;
     uct_iface_mpool_config_t       rx_mpool;
     ucs_range_spec_t               port_range;
-    int                            loopback_enable;
     struct {
         ucs_time_t                 idle;
         unsigned                   cnt;
         ucs_time_t                 intvl;
     } keepalive;
 } uct_tcp_iface_config_t;
 
+/**
+ * TCP md configuration
+ */
+typedef struct uct_tcp_md_config {
+    uct_md_config_t super;
+    uint8_t  loopback_enable;
```

Comment:
should be int

Reply:
https://github.com/openucx/ucx/pull/6720#discussion_r622269161

Reply:
my previous comment was a mistake, since it's config parser structure it has to be int